extensions/sar/README.md

```diff
@@ -30,12 +30,16 @@ To describe frame start and end times, use the [Date and Time Range fields](../.
 | sar:product_type            | string    | **REQUIRED.** The product type, for example `SSC`, `MGD`, or `SGC` |
 | sar:resolution_range        | number    | The range resolution, which is the maximum ability to distinguish two adjacent targets perpendicular to the flight path, in meters (m).  |
 | sar:resolution_azimuth      | number    | The azimuth resolution, which is the maximum ability to distinguish two adjacent targets parallel to the flight path, in meters (m).  |
-| sar:pixel_spacing_range     | number    | The range pixel spacing, which is the distance between adjacent pixels perpendicular to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type `GRD`. |
-| sar:pixel_spacing_azimuth   | number    | The azimuth pixel spacing, which is the distance between adjacent pixels parallel to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type `GRD`. |
+| sar:pixel_spacing_range     | number    | The range pixel spacing, which is the distance between adjacent pixels perpendicular to the flight path, in meters (m). Strongly RECOMMENDED to be specified for 'ground-range detected' products. |
+| sar:pixel_spacing_azimuth   | number    | The azimuth pixel spacing, which is the distance between adjacent pixels parallel to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type 'ground-range detected' products. |
+| sar:product_pixel_spacing   | number    | The actual product pixel (column) spacing, in meters (m). While `sar:pixel_spacing_range` is given in in radar geometry, this property is in ground geometry. The two properties are usually mutually exlusive. |
+| sar:product_line_spacing    | number    | The actual product line (row) spacing, in meters (m). While `sar:pixel_spacing_azimuth` is given in in radar geometry, this property is in ground geometry.  The two properties are usually mutually exlusive. |
 | sar:looks_range             | number    | Number of range looks, which is the number of groups of signal samples (looks) perpendicular to the flight path. |
 | sar:looks_azimuth           | number    | Number of azimuth looks, which is the number of groups of signal samples (looks) parallel to the flight path. |
 | sar:looks_equivalent_number | number    | The equivalent number of looks (ENL). |
 | sar:observation_direction   | string    | Antenna pointing direction relative to the flight trajectory of the satellite, either `left` or `right`. |
+| sar:measurement_type        | string    | The product's type of measurement. One of `beta0`, `gamma0` or `sigma0`. |
+| sar:measurment_convention   | string    | The product's measurement convention. One of `amplitude` (linear amplitude), `power` (linear power) or `angle`. |
```

Comment:
These are specific to SAR *amplitude* data, so should probably be in the CARD4L-NRB? There are many other SAR products out there where these terms are irrelevant (such as phase, coherence, interferometric phase)

Reply:
or rename as 'amplitude_type' and 'amplitude_convention'? note also the current typo in measurement_convention

Reply:
Yeah, I'm thinking about splitting SAR a bit into Commons, Backscatter, GRD, POL, ...

Reply:
`amplitude` is a bit of a confusing term for me, when it can be provided as `amplitude` or `power/intensity`. I understand this is to differentiate from phase and other measurement. I do prefer the terminology we use for product type in card4l, so maybe `backscatter_measurement_type` and `backscatter_measurement_convention` (or without `_measurement`)? or split into different key parameters for different product types. I think these are critical SAR relevant parameters but don't have strong view which extension they should be in.

Reply:
@fangfy The CARD4L SAR extension also includes POL, which has a list of these values, see the comment above: https://github.com/radiantearth/stac-spec/pull/962#issuecomment-772498604 - I'm just trying to align them somehow. That's why I did not use backscatter in the name.

Reply:
Thanks @m-mohr! The POL document defines two possible type of products and uses `Measurement Type` to distinguish between CovMat and PRD products. However, for the `measurement_type` discussed here, it should be gamma0.  So you are right, this field is common. But something else is needed to identify CovMat or PRD (maybe within product type). Measurement convention (unit) has angle because for PRD, there is an angle measurement. So in this case, the field is extended to identify what (image/asset specific) measurement it is. And this means, it can be extended further to include other complex/phase type of measurement.... I'm not sure I can provide a complete list though as I'm less familiar with other SAR products and terminologies people use. Maybe worth sharing current state with more people in CEOS CARD4L working group and see how best to proceed from here....

Reply:
I've changed product_type in the CARD4L extension in a way that it makes clear whether it's NRB, COVMAT or PRD. In the general SAR extension, it's (for now) up to the user to make the product_type explicit enough.

Yes, the measurement type would likely need to be extended. I guess it's fine that we just start a list and add to it later. For now, I made the set of values to a list of examples and we can add definitive (enum-like) values later.

I've also fixed the typo in measurement_convention.

Reply:
I’ll bring this up in the WG but it might take some time to get feedback.

Reply:
Thank you, @fangfy.